src/regressor.jl

```diff
@@ -65,7 +65,7 @@ function collate(model::Union{NeuralNetworkRegressor, MultivariateNeuralNetworkR
     row_batches = Base.Iterators.partition(1:length(y), batch_size)
 
     Xmatrix = MLJModelInterface.matrix(X)'
-    if y isa AbstractVector{<:Tuple}
+    if y isa Tables.MatrixTable
         return [(Xmatrix[:, b], ymatrix[:, b]) for b in row_batches]
```

Comment:
This should be `if Tables.istable(y)` not `if y isa Tables.MatrixTable`, which is too restrictive. For, examlple, user is allowed to send a DataFrame.